shared/settings/feedback-container.desktop.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+// Not implemented
+const FeedbackContainer = () => null
```

Comment:
the old feedback-container was just native only so lets just enforce that at the container level